python_cdk/src/detectors/python_cdk_kms_backing_key_rotation_enabled/python_cdk_kms_backing_key_rotation_enabled_non-compliant.py

```diff
@@ -0,0 +1,15 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-cdk-kms-backing-key-rotation-enabled@v1.0 defects=1}
+from aws_cdk.aws_kms import Key, KeySpec 
+import aws_cdk as cdk
+from aws_cdk import Stack
+
+class CdkStarterStack(cdk.Stack):
+        def __init__(self, scope: cdk.App, id: str):
+                super(scope, id)        
```

Comment:
The indentation here is not consistent as compared to the other files. Please fix.

Reply:
In general please make sure all files follow the same tab spacing for indents.